### ISSUE TITLE
TypeError if resource has no Properties defined

### ISSUE THREAD

**Issue by kristofwillen** [kristofwillen/cfntagger#2]
This cfn snippet is valid cfn:
```yaml
Resources:
  PipelineS3Bucket:
    Type: AWS::S3::Bucket
```

Running cfntagger on it produces a TypeError:
```python
[/tmp/foo.yml][Resource] PipelineS3Bucket => AWS::S3::Bucket
Traceback (most recent call last):
  File "/home/kristof/.local/bin/cfntagger", line 65, in <module>
    cfn_tagger.tag()
  File "/home/kristof/.local/lib/python3.10/site-packages/cfntagger/cfntagger.py", line 206, in tag
    if "Tags" in self.resources[item].get("Properties"):
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by kristofwillen** [comment]
created bug/noproperties branch

**Comment by kristofwillen** [comment]
Fixed in v0.9.4
